client/apps/city_issues/migrations/0001_initial.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
```

Comment:
How this migration is related to media URLs handling?

Reply:
fixed